### PR TITLE
Measure documentation

### LINKED ISSUE #1518: Automate measure documentation (readme.md.erb)
Automate the creation of documentation for each measure. Essentially, in addition to the machine-readable `measure.xml` file, there could be a human-readable `README.md` (or similar) file.

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Closes #1518. Adds measure README.md documentation. Automatically generated from README.md.erb templates when running `openstudio tasks.rb update_measures`.

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [x] ~Schematron validator (`EPvalidator.xml`) has been updated~
- [x] ~Sample files have been added/updated (via `tasks.rb`)~
- [x] ~Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests` and/or `workflow/tests/hpxml_translator_test.rb`)~
- [x] ~Documentation has been updated~
- [x] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] ~No unexpected changes to simulation results of sample files~

### CODE REVIEW

File: ReportSimulationOutput/README.md.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<br/>
+<% end %>
+
+<% if arguments.size == 0 %>
+<%= "This measure does not have any user arguments" %>
+<% end %>
+
+<% if outputs.size > 0 %>
+## Outputs
+Measure outputs are:
```
Comment (joseph-robertson) [inline ReportSimulationOutput/README.md.erb:36]: Clarify that some of these are optional?

File: ReportUtilityBills/README.md.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<br/>
+<% end %>
+
+<% if arguments.size == 0 %>
+<%= "This measure does not have any user arguments" %>
+<% end %>
+
+<% if outputs.size > 0 %>
+## Outputs
+Measure outputs are:
```
Comment (joseph-robertson) [inline ReportUtilityBills/README.md.erb:36]: This makes me think: should we have an `outputs` method for this measure?

File: ReportSimulationOutput/README.md.erb (hunk 1 above)
Comment (shorowit) [reply ReportSimulationOutput/README.md.erb:36]: Good idea, will address.

File: ReportUtilityBills/README.md.erb (hunk 2 above)
Comment (shorowit) [reply ReportUtilityBills/README.md.erb:36]: Hmm, I looked at implementing it but I don't think we actually can. All of the outputs are [specific to utility bill scenarios](https://github.com/NREL/OpenStudio-HPXML/blob/dd8a2ce30246ac57e44f21a08113f75d60c5298f/ReportUtilityBills/measure.rb#L333-L340).

Comment (joseph-robertson) [reply ReportUtilityBills/README.md.erb]: So, similarly, emissions outputs for ReportSimOutput are excluded from the `outputs` method?

Why is B true?

Comment (shorowit) [reply ReportUtilityBills/README.md.erb]: They wouldn't be additional, they'd be changing. We already register outputs with the runner, so these names would change. The only thing we don't do is populate them in the outputs section of the measure. So, for example, you can today in PAT retrieve the utility bill outputs but you have to type them in, whereas if the outputs section is available, you could choose from a dropdown menu.

Comment (joseph-robertson) [reply ReportUtilityBills/README.md.erb]: Hmm I'm confused then. Outputs in the `outputs` method couldn't be dynamically populated in the same way they are for the runner?

Comment (shorowit) [reply ReportUtilityBills/README.md.erb]: I'll create an issue around this, but I don't want to tackle it as part of this PR.

Comment (shorowit) [reply ReportUtilityBills/README.md.erb]: https://github.com/NREL/OpenStudio-HPXML/issues/1522

**Review by joseph-robertson — APPROVED** [review]

**Pull request merged**